sat/engine/testdata/tinygo-cache/main.go

```diff
@@ -1,8 +1,8 @@
 package main
 
 import (
-	"github.com/suborbital/e2core/sat/api/tinygo/runnable"
```

Comment:
Is the cache functionality surviving the purge?

Reply:
I don't think any of these TinyGo modules compile with that import. The library still lives in the Reactr repo.

Reply:
and yeah cache is going away

Reply:
Haha no, I missed removing this one